app/scripts/templates/sign_up_password.mustache

```diff
@@ -0,0 +1,54 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-signup-password-header">
+      {{#serviceName}}
+        <!-- L10N: For languages structured like English, the second phrase can read "to continue to %(serviceName)s" -->
+        {{#t}}Create a Firefox Account{{/t}} <span class="service">{{#t}}Continue to %(serviceName)s{{/t}}</span>
+      {{/serviceName}}
+      {{^serviceName}}
+        {{#t}}Create a Firefox Account{{/t}}
+      {{/serviceName}}
+    </h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success"></div>
+
+    <form novalidate>
+      <div class="input-row">
+        <input name="email" type="email" class="email back" value="{{ email }}" />
+      </div>
+
+      <div class="input-row password-row">
+        <input id="password" type="password" class="password check-password tooltip-below" placeholder="{{#t}}Password{{/t}}" value="{{ password }}" pattern=".{8,}" required autofocus data-form-prefill="password" data-synchronize-show="true" />
```

Comment:
I found that if I put a password here, then go back to email (re-render), put a different email in, go back to sign_up then the original password still stays. feature or bug?